interface/main/calendar/find_group_popup.php

```diff
@@ -0,0 +1,306 @@
+            cursor: hand;
+            cursor: pointer;
+        }
+
+        #searchResults td {
+            font-size: 0.7em;
+            border-bottom: 1px solid #eee;
+        }
+
+        .oneResult {
+        }
+
+        .billing {
+            color: red;
+            font-weight: bold;
+        }
+
+        /* for search results or 'searching' notification */
+        #searchstatus {
+            font-size: 0.8em;
+            font-weight: bold;
+            padding: 1px 1px 10px 1px;
+            font-style: italic;
+            color: black;
+            text-align: center;
+        }
+
+        .noResults {
+            background-color: #ccc;
+        }
+
+        .tooManyResults {
+            background-color: #fc0;
+        }
+
+        .howManyResults {
+            background-color: #9f6;
+        }
+
+        #searchspinner {
+            display: inline;
+            visibility: hidden;
+        }
+
+        /* highlight for the mouse-over */
+        .highlight {
+            background-color: #336699;
+            color: white;
+        }
+    </style>
+
+    <script type="text/javascript"
+            src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-2-2/index.js"></script>
+    <!-- ViSolve: Verify the noresult parameter -->
+    <?php
+    if (isset($_GET["res"])) {
+        echo '
+<script language="Javascript">
+			// Pass the variable to parent hidden type and submit
+			opener.document.theform.resname.value = "noresult";
+			opener.document.theform.submit();
+			// Close the window
+			window.self.close();
+</script>';
+    }
+    ?>
+    <!-- ViSolve: Verify the noresult parameter -->
+
+    <script language="JavaScript">
+
+        function selgid(gid, name) {
+            if (opener.closed || !opener.setgroup)
+                alert("<?php echo htmlspecialchars(xl('The destination form was closed; I cannot act on your selection.'), ENT_QUOTES); ?>");
+            else
+                opener.setgroup(gid, name);
+            window.close();
+            return false;
+        }
+
+    </script>
+
+</head>
+
+<body class="body_top">
+
+<div id="searchCriteria">
+    <form method='post' name='theform' id="theform"
+          action='find_group_popup.php?<?php if (isset($_GET['pflag'])) echo "pflag=0"; ?>'>
+        <?php echo htmlspecialchars(xl('Search by:'), ENT_NOQUOTES); ?>
+        <select name='searchby'>
+            <option value="Name"><?php echo htmlspecialchars(xl('Name'), ENT_NOQUOTES); ?></option>
+            <option
+                value="ID"<?php if ($searchby == 'ID') echo ' selected' ?>><?php echo htmlspecialchars(xl('ID'), ENT_NOQUOTES); ?></option>
+        </select>
+        <?php echo htmlspecialchars(xl('for:'), ENT_NOQUOTES); ?>
+        <input type='text' id='searchparm' name='searchparm' size='12'
+               value='<?php echo htmlspecialchars($_REQUEST['searchparm'], ENT_QUOTES); ?>'
+               title='<?php echo htmlspecialchars(xl(''), ENT_QUOTES); ?>'>
```

Comment:
this is very weird to have an empty xl???. Isn't that the same as just not having a title attribute?